From 766963a147cbd19a9f7b3192a5fac48325a85039 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Wed, 22 May 2019 14:28:37 -0700
Subject: [PATCH 35/44] Revert "NightDisplayController: report unavailable if
 livedisplay feature is present"

This reverts commit 5ef7a9bb0996cc0c141d054514744ceaa93ba036.

Change-Id: I4a23293088b65b62185002701113a62f32d495c6
---
 .../com/android/internal/app/ColorDisplayController.java    | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/core/java/com/android/internal/app/ColorDisplayController.java b/core/java/com/android/internal/app/ColorDisplayController.java
index a85457e630e..ba6cf269b73 100644
--- a/core/java/com/android/internal/app/ColorDisplayController.java
+++ b/core/java/com/android/internal/app/ColorDisplayController.java
@@ -558,12 +558,10 @@ public final class ColorDisplayController {
     }
 
     /**
-     * Returns {@code true} if Night display is supported by the device,
-     * unless LiveDisplay feature is available.
+     * Returns {@code true} if Night display is supported by the device.
      */
     public static boolean isAvailable(Context context) {
-        return context.getResources().getBoolean(R.bool.config_nightDisplayAvailable) &&
-                !context.getPackageManager().hasSystemFeature("org.lineageos.livedisplay");
+        return context.getResources().getBoolean(R.bool.config_nightDisplayAvailable);
     }
 
     /**
-- 
2.17.1

